vizro-core/tests/unit/vizro/models/test_dashboard.py

```diff
@@ -156,6 +156,30 @@ def test_page_registry_with_image(self, page_1, mocker, tmp_path, image_path):
             layout=mocker.ANY,  # partial call is tricky to mock out so we ignore it.
         )
 
+    @pytest.mark.parametrize(
+        "image_path",
+        [
+            "logo.svg",
+            "logo.png",
+            "logo.apng",
+            "logo.avif",
+            "logo.gif",
+            "logo.jpeg",
+            "logo.jpg",
+            "logo.webp",
+            "images/logo.svg",
```

Comment:
I don't remember, but does this work? If not, we should add it to the docs that the logo image needs to be in the root folder or one level lower max.

```suggestion
            "images/icons/logo.svg",
```

Reply:
This will work - it can be any level inside `assets`.

Reply:
(so no need to make any changes here @nadijagraca)